tests/puppeteer/.docker/prestashop/ps_git

```diff
@@ -0,0 +1,29 @@
+FROM prestashop/prestashop-git
```

Comment:
rename in Dockerfile

Reply:
Do mean to have only one DockerFile or to just change extension ? (because we already have a Dockerfile used to import local version to docker container)

Reply:
as we said in private. juste rename in `Dockerfile-from-git` or something similar

Reply:
Done thanks